### PR TITLE
Try 4110 for file_permissions_sudo

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:
Try 4110 for file_permissions_sudo.

#### Rationale:
Better align with the benchmark.

### CODE REVIEW

**Comment by vojtapolasek** [comment]
I think it might be worth modifying anssi.yml control and move the rule from related_rules to rules... and let's see what the tests show.

**Comment by vojtapolasek** [comment]
hm... I did a thorough investigation of this requirement and I wonder how to solve it.
I see two things which we should consider.
The first is related to the file_permissions_sudo rule. I think we should set the allow_stricter_permissions to false, because stricter permissions also means for example no suid, that defends purpose of sudo. But in general, I think this rule can be in the profile and by that I mean in the list of active rules, not related rules.
The second thing which I noticed is that we explicitly unselect the sudo_dedicated_group for RHEL 9 version of the profile... I think we should reconsider it. Or do you know the history of the reason why the rule was not added to RHEL 9?
Anyway, the requirement can't be fully automated exactly because of the sudo_dedicated_group rule, this has to be remediated manually.

File: controls/anssi.yml [hunk 1]
```diff
@@ -907,11 +907,10 @@ controls:
 
       The file_permissions_sudo rule is in related_rules rules as the required permissions of 4750 is not
       fully satisfied. See https://github.com/ComplianceAsCode/content/issues/11779.
-    status: partial
+    status: automated
```
Comment (marcusburghardt) [inline controls/anssi.yml:910]: Could you also remove the notes informing the requirement is not fully satisfied, please?

**Review by marcusburghardt — CHANGES_REQUESTED** [review]
Just a minor point, to remove an outdated note.

File: controls/anssi.yml (hunk 1 above)
Comment (Mab879) [reply controls/anssi.yml:904]: Done.

**Review by marcusburghardt — APPROVED** [review]
Thanks

**Pull request merged**